components/tooltip/index.jsx

```diff
@@ -98,17 +95,29 @@ const propTypes = {
 	 */
 	triggerStyle: PropTypes.object,
 	/**
-	 * Determines the variant of tooltip: for informative purpose (blue background) or warning purpose (red background)
+	 * Determines the theme of tooltip: for informative purpose (blue background) or warning purpose (red background). This used to be `variant`.
+	 */
+	theme: PropTypes.oneOf(['info', 'error']),
+	/**
+	 * Determines the type of the tooltip.
 	 */
-	variant: PropTypes.oneOf(['info', 'error']),
+	variant: PropTypes.oneOf(['base', 'learnMore']),
 };
 
 const defaultProps = {
+	assistiveText: {
+		learnMore: 'this link',
+	},
 	align: 'top',
 	content: <span>Tooltip</span>,
+	labels: {
+		learnMoreAfter: 'to learn more.',
+		learnMoreBefore: 'Click',
+	},
 	hoverCloseDelay: 50,
 	position: 'absolute',
-	variant: 'info',
+	theme: 'info',
+	variant: 'base',
 };
 
 /**
```

Comment:
We should rename this class to just Tooltip within this file.

Reply:
Done.